__tests__/components/dashtable_v2.test.jsx

```diff
@@ -0,0 +1,161 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import GlobalDashboardTable from '../../components/dashtable_v2.js';
+
+describe('GlobalDashboardTable', () => {
+  test('renders dashtable with correct data format', () => {
```

Comment:
Can you write a more descriptive name for the test, something more specific than simply "correct".  In this case, since your test data has two students, you could write something like, "renders dashtable with two student rows" (if I'm understanding the test case correctly).